alot/helper.py

```diff
@@ -631,3 +631,29 @@ def email_as_string(mail):
                            as_string, flags=re.MULTILINE)
 
     return as_string
+
+
+class classproperty(object):
+    """Decorator to emulate 'class properties' using class level methods.
+    
+    The decorator is applied to a method to make it a class property. E.g.
+    
+    .. code-block:: python
```

Comment:
I using 

    ``` python
    stuff
    ```

in the docstring triggered an error so I am using `.. code-block:: python` instead. Was I correct to do so?